commit 6d2f1763376d6bc5611af5226440d1583697c162
Author: Joe Thornber <ejt@redhat.com>
Date:   Thu Feb 9 11:46:18 2017 -0500

    dm cache: fix corruption seen when using cache > 2TB
    
    commit ca763d0a53b264a650342cee206512bc92ac7050 upstream.
    
    A rounding bug due to compiler generated temporary being 32bit was found
    in remap_to_cache().  A localized cast in remap_to_cache() fixes the
    corruption but this preferred fix (changing from uint32_t to sector_t)
    eliminates potential for future rounding errors elsewhere.
    
    Signed-off-by: Joe Thornber <ejt@redhat.com>
    Signed-off-by: Mike Snitzer <snitzer@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1262_6d2f1763376d6bc5611af5226440d1583697c162.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1262_6d2f1763376d6bc5611af5226440d1583697c162.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
